utm.yaml

```diff
@@ -1524,6 +1582,13 @@ paths:
           description: |-
             * The provided key did not prove knowledge of all current and relevant airspace Entities
             * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+        '412':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          description: |-
+            The client attempted to transition the Operation to Accepted or Activated while marked as Down in the DSS.
```

Comment:
Should discuss whether a USS should be able to make ANY state changes when marked Down in the DSS? If marked down because it isn't sending p2p notifications, then it might change the state in the DSS without notifying all subscribers.

Reply:
Certainly we should discuss anything important.  My thought is that NonConforming and Contingent are not planned states; they are used to communicate what is actually happening.  If a USS isn't responding properly (and therefore is marked down) and then has a flight go contingent, it seems better to at least have a record in the DSS indicating that contingency (even if the details aren't accessible and even if notifications aren't sent properly) than for other USSs to have no indication of the contingency (which would happen if the down USS were not allowed to write their Contingent operation to the DSS).

Reply:
Ok. This also means the USS can end the flight while USS is down. We probably need a specific shall statement in the standard requiring this behavior of the DSS.

Reply:
It seems like deleting an Operation should probably not be allowed either as we want to maximize information available in off-nominal cases -- I'll make that change.